config/module.config.php

```diff
@@ -6,6 +6,10 @@
         ],
     ],
     'phpab' => [
+        'analytics' => [
+            'collector' => null,
```

Comment:
By default we don't configure a data collector and renderer. We leave this up to user-land. Maybe the user wants to use PDO analytics?

We could argue to set this up by default. I'm up for suggestions.